addons/pos_self_order/views/product_views.xml

```diff
@@ -1,19 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="product_template_search_view_pos" model="ir.ui.view">
-        <field name="name">product.template.search.pos.form</field>
-        <field name="model">product.template</field>
-        <field name="inherit_id" ref="point_of_sale.product_template_search_view_pos"/>
-        <field name="arch" type="xml">
-            <filter name="filter_to_availabe_pos" position="after">
-                <filter name="filter_to_self_order" string="Available in Self" domain="[('self_order_available', '=', True)]"/>
-            </filter>
-            <filter name="filter_to_self_order" position="after">
-                <filter name="filter_to_not_available_pos" string="Not available in Self" domain="[('self_order_available', '=', False)]"/>
-             </filter>
-        </field>
-    </record>
```

Comment:
Instead of removing this, isn't there a way of showing those filters only when you come from the pos application and open the product so that, when you open the product view from the inventory app you do not see the filter but when you open it from the pos app you can chose them ?

Reply:
Hello @adgu-odoo 
Yes, it is possible. I have updated the code and visible it from pos only

Thanks.